src/status_im2/contexts/wallet/common/collectibles/view.cljs

```diff
@@ -0,0 +1,27 @@
+(ns status-im2.contexts.wallet.common.collectibles.view
```

Comment:
nit, should we call it `(ns status-im2.contexts.wallet.common.collectibles-tab.view` ? 
i.e change it to collectibles-tab? 🤔